RELEASENOTES/v2.0.0.alpha-3.MD

```diff
@@ -0,0 +1,61 @@
+# Notary v2 Alpha 3 Release
+
+## Overview
+Notary v2 working group is pleased to announce the third release, Alpha 3. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
+- [Notary v2 specifications](https://github.com/notaryproject/notaryproject) ***[(notaryproject-v1.0.0-draft.3)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-draft2)***    (Updated in this release)  
+- [Notation Library 1](https://github.com/notaryproject/notation-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-go-lib/releases/tag/v0.7.0-alpha.1)***   (Updated in this release)
+- [Notation Library 2](https://github.com/notaryproject/notation-core-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-go-lib/releases/tag/v0.7.0-alpha.1)***   (Added in this release)
```

Comment:
the link for notation-core-go: https://github.com/notaryproject/notation-core-go/tree/v0.1.0-alpha.3